src/app/handlers/flightsql.rs

```diff
@@ -91,14 +91,12 @@ pub fn normal_mode_handler(app: &mut App, key: KeyEvent) {
                                                     Ok(batch) => {
                                                         info!("Batch rows: {}", batch.num_rows());
                                                         batches.push(batch);
-                                                        break;
```

Comment:
the complaint is that this break means the outer loop stops

I think the `break`s here mean effectively this code would only ever retrieve the first batch from the remote flightsql server which doesn't seem like the actual intent

Reply:
it actually was the intent but only as a short term hack to prevent queries with large number of results from taking a long time to return result.  i still need to implement auto pagination logic.

Reply:
since this is flightsql and no dataframe to work with i had in mind auto appending `LIMIT` and `OFFSET` to the queries.  do you have any other ideas?

Reply:
I suggest:
1. initially just signal somehow that the stream is truncated and only a part is shown (we can change the code to make this explicit with a comment to get clippy happy)
2. eventually, I can imagine some much fancier logic that fetches a stream incrementally as the user wants to page through the results (if they do). That would only buffer the rows that haev actually been shown, but would allow the user to scroll through them efficiently

I don't think there is any reason to use `LIMIT` / `OFFSET` unless there are a huge number of rows and the offset is large (like `OFFSET 1000000` or something). I think handling it on the client side would be a lot easier

Reply:
I thought i had gotten the LIMIT / OFFSET idea from dbeaver but I just double checked and im not seeing it there so yes i agree it doesnt make sense.  My original idea was to just paginate through the stream which I believe is same as your second point so I think were aligned on approach here.

Reply:
related dicussion https://github.com/datafusion-contrib/datafusion-tui/issues/137